### PR TITLE
Adding configuration example for Improv@LCRC

### PR DESCRIPTION
**Pull request by yadudoc**
# Description

This PR adds a configuration example for Improv at Argonne's LCRC.

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Update to human readable text: Documentation/error messages/comments

### CODE REVIEW

File: parsl/configs/improv.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                provider=PBSProProvider(
+                    account="YOUR_ALLOCATION_ON_IMPROV",
+                    # PBS directives (header lines): for array jobs pass '-J' option
+                    # scheduler_options='#PBS -J 1-10',
+                    scheduler_options='',
+
+                    queue="compute",
+                    # Command to be run before starting a worker, such as:
+                    # 'module load Anaconda; source activate parsl_env'.
+                    # **WARNING** Improv requires an openmpi module to be
```
Comment (benclifford) [inline parsl/configs/improv.py:20]: maybe the example here should include loading an openmpi module, either in the explicit value or in the comment on this line

File: parsl/configs/improv.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+from parsl.providers import PBSProProvider
+
+config = Config(
+        executors=[
+            HighThroughputExecutor(
+                label="Improv_multinode",
+                max_workers_per_node=32,
+                provider=PBSProProvider(
+                    account="YOUR_ALLOCATION_ON_IMPROV",
+                    # PBS directives (header lines): for array jobs pass '-J' option
```
Comment (benclifford) [inline parsl/configs/improv.py:13]: i don't think we should be encouraging people to use array jobs. i've only ever seen one dubious need for them.

File: parsl/configs/improv.py (hunk 1 above)
Comment (benclifford) [reply parsl/configs/improv.py:20]: oh you did, i misread `'''` as `''`

File: parsl/configs/improv.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                    # scheduler_options='#PBS -J 1-10',
+                    scheduler_options='',
+
+                    queue="compute",
+                    # Command to be run before starting a worker, such as:
+                    # 'module load Anaconda; source activate parsl_env'.
```
Comment (benclifford) [inline parsl/configs/improv.py:19]: there's already an example so you don't need this second example?

Comment (yadudoc) [reply parsl/configs/improv.py]: Fair enough, I've removed this one.

File: parsl/configs/improv.py (hunk 2 above)
Comment (yadudoc) [reply parsl/configs/improv.py]: Removed.

**Review by benclifford — APPROVED** [review]

**Pull request merged**